docs/reference/settings.rst

```diff
@@ -206,6 +206,42 @@ when :setting:`ZYTE_API_LOG_REQUESTS` is enabled, excluding object keys.
 To disable truncation, set this to ``0``.
 
 
+.. setting:: DOWNLOAD_MAXSIZE
+
+DOWNLOAD_MAXSIZE
+================
```

Comment:
I don't think we need to copy the documentation here, we should just mention that these two standard Scrapy settings are supported.